docs/en/setup/backend/configuration-vocabulary.md

```diff
@@ -114,6 +114,7 @@ The Configuration Vocabulary lists all available configurations provided by `app
 | -                       | -             | profileDataQueryScrollBatchSize                                                                                                                                          | The batch size of query profiling data.                                                                                                                                                                                                                                                                                                                                                                                                                         | SW_STORAGE_ES_QUERY_PROFILE_DATA_BATCH_SIZE       | 100                                                                     |
 | -                       | -             | advanced                                                                                                                                                                 | All settings of ElasticSearch index creation. The value should be in JSON format.                                                                                                                                                                                                                                                                                                                                                                               | SW_STORAGE_ES_ADVANCED                            | -                                                                       |
+| -                       | -             | logicSharding                                                                                                                                          | If disabled, all metrics will persist in one index, to reduce the number of indexes. If enabled, the metrics will persist in multi indexes according to their index name and aggregation function name,the metrics which has the same aggregation function name will persist in one index.                                                                                                                                                                                        | SW_STORAGE_ES_LOGIC_SHARDING       | false                                                                     |
```

Comment:
Please update the description according to the doc update. Usually, the document should describe things from `enable` perspective. Change log would focus on the impact and how to fall back.